packages/components/src/advanced-filters/example.md

```diff
@@ -0,0 +1,140 @@
+```jsx
+import { AdvancedFilters } from '@woocommerce/components';
+const { orderStatuses } = wcSettings;
+
+const path = '';
+const query = {};
```

Comment:
If you place `path: '/devdocs'` in the query, the devDocs example will actually update the value when you click "filter", otherwise it goes to Dashboard.